internal/core/operations/healthcontroller/health_controller_executor.go

```diff
@@ -200,12 +200,12 @@ func (ex *SchedulerHealthControllerExecutor) findAvailableAndExpiredRooms(ctx co
 		}
 
 		switch {
-		case ex.isRoomStatus(room, game_room.GameStatusPending):
-			availableRoomsIDs = append(availableRoomsIDs, gameRoomID)
 		case ex.isRoomStatus(room, game_room.GameStatusUnready):
 			availableRoomsIDs = append(availableRoomsIDs, gameRoomID)
```

Comment:
Since the logic seems to test if the room is expired somehow, and the non-expired are considered available, why do we need to check if the status is unready? Seems to me that we could take off the unready condition 🤔

Reply:
We have to consider unready rooms in the initialization timeout check as well!